stdlib/public/core/Optional.swift

```diff
@@ -167,7 +196,37 @@ public enum Optional<Wrapped>: ExpressibleByNilLiteral {
       return .none
     }
   }
+}
+
+extension Optional where Wrapped: ~Copyable {
+  @_alwaysEmitIntoClient
```

Comment:
You added `@inlinable` to the `flatMap` variants of these, but not here...?

Reply:
Grr, that was a copy/paste error. `@inlinable` and `@_alwaysEmitIntoClient` should never both be present.